.github/workflows/test.yaml

```diff
@@ -19,11 +19,14 @@ jobs:
     name: Run tests defined for the project
     runs-on: ubuntu-latest
     container:
-      image: ubuntu:18.04
+      image: ubuntu:20.04
     steps:
       - name: Setup ubuntu container
         run: |
           apt update
+          export TZ=Etc/Utc
+          ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+          export DEBIAN_FRONTEND=noninteractive
```

Comment:
in my experience, github workflows work better if instead of `export`, we do one of:
```sh
echo TZ=Etc/UTC >> $GITHUB_ENV
```
or specify it in the job itself:
```yaml
env:
  TZ: "Etc/UTC"
```

Reply:
I have "fixed" this by removing it as part of a larger change to moving from `ubuntu-latest` with `ubuntu:18.04` container specified to using `runs-on: ubuntu-20.04`.